docs/content/en/usage.md

```diff
@@ -24,20 +24,22 @@ See [ky](https://github.com/sindresorhus/ky#options) docs for all available opti
 **Example: GET JSON data**
 
 ```js
-const res = await $http.get('https://unpkg.com/nuxt/package.json')
-const data = await res.json()
+const data = await $http.$get('https://unpkg.com/nuxt/package.json')
 ```
 
-Alternatively for json only you can use `$` prefixed shortcut that smartly parses response using [destr](https://github.com/nuxt-contrib/destr).
+In most of the case, you want to get the JSON response. You can use `$` prefixed shortcut that smartly parses response using [destr](https://github.com/nuxt-contrib/destr).
+
 
+Alternatively for other response type, you can use the methods mentioned above :
```

Comment:
Here we can say it will smart parse the answer when text is returned, example:

```js
const ip = await this.$http.$get('http://icanhazip.com')
```

Reply:
In this case, i would like purpose to convert response stream into another type (for example text or someone else) if it's ok for you ?

Reply:
Good idea for stream response!

Reply:
Great idea, i will try to purpose an example of stream response 👍